models/genre.rb

```diff
@@ -0,0 +1,13 @@
+require './item'
+
+class Genre < Item
+  attr_reader :id
+  attr_accessor :name, :items
+
+  def initialize(name)
+    super
+    @id = Random.rand(1...1000)
+    @name = name
+    @items = []
+  end
+end
```

Comment:
- [ ] Kindly implement the association between the genre and the item class as stated in the project requirements.
- [ ] The Genre class is not supposed to inherit the item properties. kindly remove that inheritance.

Reply:
Noted. I have fixed it.

Reply:
:rocket: :+1: :partying_face: